csrc/python_frontend/python_bindings.cpp

```diff
@@ -3266,5 +3306,60 @@ void initNvFuserPythonBindings(PyObject* module) {
             !isResharding(tv->definition()));
       },
       py::arg("tensor"));
+  nvf_sched.def(
+      "can_schedule",
+      [](FusionDefinition::SchedOperators& self,
+         const ScheduleHeuristic& heuristic) {
+        NVF_CHECK(
+            self.validUse(),
+            "Attempting to use a SchedOperators Op prior to definition!");
+        return canSchedule(self.fusion_definition->userSchedule(), heuristic);
+      },
+      py::arg("heuristic"));
+  nvf_sched.def("which_schedulers", [](FusionDefinition::SchedOperators& self) {
```

Comment:
The name is not very clear to me. Something like `fd.sched.supported_schedulers()` might be better?

Reply:
I changed it to `find_compatible_schedulers`